Dockerfile

```diff
@@ -1,26 +1,24 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3-298.1618432845
+FROM registry.access.redhat.com/ubi9:9.2-755
```

Comment:
If there are no important dependency requirement, I still recommend use the UBI minimal image. 

| Name         | Size    | Uncompressed Size |
|--------------|---------|-------------------|
| ubi9         | 74.5 MB | 206.9 MB          |
| ubi9-minimal | 36.1 MB | 92.9 MB           |

Less size means less code and less security leak. 
I think ubi8-mininal has been verified all features we need are supported.

PS: I have't check if there has any duplicate packages had been installed in ubi base image, but will install again along with httpd installation.

IMO, this is not a big problem, but we should choose the better one for our situation